config/prod.exs

```diff
@@ -36,6 +35,16 @@ config :ueberauth, Ueberauth.Strategy.Cognito,
   user_pool_id: {System, :get_env, ["COGNITO_USER_POOL_ID"]},
   aws_region: {System, :get_env, ["COGNITO_AWS_REGION"]}
 
+if System.get_env("ENVIRONMENT_NAME") == "dev" do
+  config :signs_ui, SignsUiWeb.Endpoint,
+    screenplay_base_url: "localhost:4000/ https://screenplay-dev.mbtace.com/"
```

Comment:
Is the `localhost:4000/` here intentional or a copy/paste error?

Reply:
@cmaddox5 it's intentional but lmk what your thoughts are. Basically, I added localhost so that when you run screenplay locally, Signs UI Dev will let it embed the PAESS renderings. Otherwise, one would need to have a local instance of Signs UI running alongside Screenplay in order for things to fully work

Reply:
Ahh I see. That makes sense. Especially for the read-only phase. When we start implementing overrides, we may want to ensure that we are not locally testing overrides on alerts outside of local development. But in the meantime, this will be helpful.